### PR TITLE
Move checking performance marks to separate file

### PR DESCRIPTION
**Pull request by mmarkelov**
I suppose may be it will be a good idea to move performance checking logic to separate file helper

### CODE REVIEW

File: src/ThemedStyleSheet.js [hunk 1]
```diff
@@ -31,22 +33,14 @@ function get() {
 }
 
 function resolve(...styles) {
-  if (
-    process.env.NODE_ENV !== 'production'
```
Comment (ljharb) [inline src/ThemedStyleSheet.js]: The tradeoff here is that now bundlers can’t delete the code, they have to always import a function and call it.

Comment (lencioni) [reply src/ThemedStyleSheet.js]: Agreed, I think we should leave this inline so that it can be completely minified out in production builds.

Comment (mmarkelov) [reply src/ThemedStyleSheet.js]: Thank you for your feedback. I wanted to keep it just a little bit cleaner. But now I understand that it was not the best approach

Comment (lencioni) [reply src/ThemedStyleSheet.js]: Thanks for the PR!

**Pull request closed without merging**